examples/platform/k32w/app/support/Memconfig.cpp

```diff
@@ -0,0 +1,180 @@
+/*
+ *
+ *    Copyright (c) 2020-2021 Project CHIP Authors
+ *    All rights reserved.
+ *
+ *    Licensed under the Apache License, Version 2.0 (the "License");
+ *    you may not use this file except in compliance with the License.
+ *    You may obtain a copy of the License at
+ *
+ *        http://www.apache.org/licenses/LICENSE-2.0
+ *
+ *    Unless required by applicable law or agreed to in writing, software
+ *    distributed under the License is distributed on an "AS IS" BASIS,
+ *    WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *    See the License for the specific language governing permissions and
+ *    limitations under the License.
+ */
+
+/**
+ *    @file
+ *      This file implements heap memory allocation APIs for CHIP. These functions are platform
```

Comment:
This doesn't actually reimplement `CHIPPlatformMemory*` but rather steers stdlib calloc/malloc/free such that the *default* stdlib-based  `CHIPPlatformMemory*` works as intended with the platform's heap. The comment reflect that, rather than being a generic cut'n'paste from `src/lib/support/CHIPMem-Malloc.cpp` comment.

Reply:
Modified comment